### DISCUSSION TITLE
Support for JDK 17

### DISCUSSION THREAD

**Discussion by ijwmiranda** [orientechnologies/orientdb#10172] [Q&A]
On the documentation, it's stated that only runs on Java 8 and 11: 

> To run OrientDB you will need a Java SDK installed on your machine. OrientDB runs with Java from Version 8 to 11.

But on the release notes of 3.2.3, it was stated that it was started the support for Java 17 but i didn't found any other reference on the latest release notes.

Do orientdb have support for Java 17 on the latest version?

Thanks in advance

**Comment by tglman** [comment]
Hi,

Yes the last 3.2.x releses support java 17 and later (our CI build do run with the jdks 8,11,17 ), I could test it successfully at least till java 19, our not released developer branch for future version do test against 17,19 and 22.

Regards
